### ISSUE TITLE
Intermittent CSP violations on Chrome

### ISSUE THREAD

**Issue by codemasterover9000** [obogo/ux-angularjs-datagrid#110]
Hi,

When accessing our webapp with fairly strict CSP settings from Chrome we intermittently see violations being logged.
Our CSP header is:
```
Content-Security-Policy:default-src 'self'; style-src 'self' 'sha256-ngbCwp+AGcmJASc+YnB8dKz0S18jFF4yJ8TWKCo7V/8='
```
The log message is:
```
VM239:1 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'sha256-ngbCwp+AGcmJASc+YnB8dKz0S18jFF4yJ8TWKCo7V/8='". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution.
```
The hash 47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU= is the encoding of an empty string.

When looking at the stack in the Chrome debugger the processCompilation function is trying to set a style attribute to an empty value, at the statement: clone.attr(attr.name, attr.value);

We're using ux-datagrid v1.6.8, and angularjs 1.6.1. Chrome is version 63.0.3239.132.

**Comment by codemasterover9000** [comment]
Seen on Firefox 58.0 too.

**Comment by wesjones** [comment]
Sorry for the long delay on this one. The fix is now in as part of 1.6.12. Thanks.

**Comment by codemasterover9000** [comment]
No problem, thanks for fixing it!
